app/src/main/java/org/wikipedia/settings/Prefs.kt

```diff
@@ -792,6 +792,10 @@ object Prefs {
         get() = PrefsIoUtil.getBoolean(R.string.preference_key_yir_survey_shown, false)
         set(value) = PrefsIoUtil.setBoolean(R.string.preference_key_yir_survey_shown, value)
 
+    var showYearInReviewSurvey
+        get() = PrefsIoUtil.getBoolean(R.string.preference_key_show_yir_survey, false)
+        set(value) = PrefsIoUtil.setBoolean(R.string.preference_key_show_yir_survey, value)
```

Comment:
I feel like we should have this saved in the `YearInReviewModel`, otherwise we will not be able to show the survey for next year's YiR.